src/NavBar.css

```diff
@@ -0,0 +1,26 @@
+.nav{
+    top: 0;
+    
+    background: rgba(0,0,0,.3);
+    box-shadow: 0 3px 6px rgba(0,0,0,.16);
+    -webkit-backdrop-filter: blur(4px);
+    backdrop-filter: blur(4px);
+    padding: 1% ;
+    width: 98%;
+    z-index: 1000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    
```

Comment:
please add the fixed property to the navbar so that it doesn't move with the page,

hint:
- position: fixed

More information about fixed behavior can be found [here](https://www.w3schools.com/howto/howto_css_fixed_menu.asp)

Reply:
if I set position fixed to nav bar, the input element is going behind and nav bar coming on the top. @KoichiMin , can you help me figure this out. @J0SAL can you be more specific about setting nav bar position fixed?
 
![image](https://user-images.githubusercontent.com/105518066/227340702-1b8e2273-68df-4618-81ad-e5d196ed1d6c.png)

Reply:
> if I set position fixed to nav bar, the input element is going behind and nav bar coming on the top. @KoichiMin , can you help me figure this out. @J0SAL can you be more specific about setting nav bar position fixed?
> 
> ![image](https://user-images.githubusercontent.com/105518066/227340702-1b8e2273-68df-4618-81ad-e5d196ed1d6c.png)

This is the expected result, 
Actually, when you set an element's position to "fixed," it takes it out of the normal document flow and positions it relative to the browser window. This means that it will no longer take up space in the normal flow of the document, and other elements will act as if it is not there.

Now if you try scrolling the page, you'll see that the navbar will remain in the same place irrespective of where you are scrolling.

To Solve the overlapping problem, you can try adding a **padding-top** property to the div element after Navbar in App.js

Reply:
@J0SAL I just set nav bar position fixed and responsive.